Plugins/Memory/Collect-MemoryWinpmem.ps1

```diff
@@ -0,0 +1,194 @@
+<#
+
+.SYNOPSIS
+    Plugin-Name: Collect-MemoryWinpmem.ps1
+    
+.Description
+    This plugin captures memory from a remote host by deploying winpmem to the 
+    remote machine and captures memory locally to the remote machine before
+    copying back to the output location for PowerResponse. This plugin will
+    remove the winpmem PE and delete all locally created files after successfully
+    copying the memory acquisition back to the output destionation in
+    Power-Response.
+
```

Comment:
Might be worth dropping a link or two off to 7za.exe and winpmem downloads? I had to root around a bit to find them and still not sure if I have the right 7za.exe since the official download site (https://www.7-zip.org/download.html) lists the single standalone exe for both x86 and x64. Also wondering if my winpmem download is correct due to the `-dd` error I'm getting.